pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -38,9 +38,10 @@ var (
 		{Action: datasources.ActionRead, Scope: datasources.ScopeAll},
 	}
 	// DashboardServiceImpl implements the DashboardService interface
-	_ dashboards.DashboardService             = (*DashboardServiceImpl)(nil)
-	_ dashboards.DashboardProvisioningService = (*DashboardServiceImpl)(nil)
-	_ dashboards.PluginService                = (*DashboardServiceImpl)(nil)
+	_                  dashboards.DashboardService             = (*DashboardServiceImpl)(nil)
+	_                  dashboards.DashboardProvisioningService = (*DashboardServiceImpl)(nil)
+	_                  dashboards.PluginService                = (*DashboardServiceImpl)(nil)
+	remainingTrashDays                                         = 24 * 30 * time.Hour
```

Comment:
the formatted looks weird here

Reply:
The name is not accurate since it is not related to the remaining days. It's related to how many days all dashboards will live in the trash before deletion.  I would look for something like 'daysInTrash', 'periodBeforeFinalDeletion', 'periodInTrash', 'trashPeriod', etc.